### PR TITLE
Create README.md

### PR DESCRIPTION
**Pull request by Ericgre**
# PR Checklist

Check these items before submitting a PR... 

[Contribution Guide](https://github.com/Azure/azure-quickstart-templates/blob/master/1-CONTRIBUTION-GUIDE/README.md)

[Best Practice Guide](https://github.com/Azure/azure-quickstart-templates/blob/master/1-CONTRIBUTION-GUIDE/best-practices.md)

- [ X] - Please check this box once you've submitted the PR if you've read through the Contribution Guide and best practices checklist.

### CODE REVIEW

**Comment by Ericgre** [comment]
This PR is for the new private endpoint feature for Azure Web App

File: 101-private-endpoint-webapp/azuredeploy.json [hunk 1]
```diff
@@ -0,0 +1,224 @@
+{
+    "$schema": "https://schema.management.azure.com/schemas/2019-04-01/deploymentTemplate.json#",
+    "contentVersion": "1.0.0.0",
+    "parameters": {
+        "virtualNetwork_name": {
```
Comment (bmoore-msft) [inline 101-private-endpoint-webapp/azuredeploy.json:5]: add metadata description for all parameters

File: 101-private-endpoint-webapp/azuredeploy.json [hunk 2]
```diff
@@ -0,0 +1,224 @@
...
+            "type": "String"
+        },
+        "site_name": {
+            "type": "String"
+        },
+        "virtualNetwork_CIDR": {
+            "type": "String",
+            "defaultValue": "10.200.0.0/16"
+        },
+        "subnet1_name": {
```
Comment (bmoore-msft) [inline 101-private-endpoint-webapp/azuredeploy.json:18]: add defaultValues for all the params where it makes sense (resource names, skus)

File: 101-private-endpoint-webapp/azuredeploy.json [hunk 3]
```diff
@@ -0,0 +1,224 @@
...
+            "location": "[parameters('location')]",
+            "dependsOn": [
+                "[resourceId('Microsoft.Web/serverfarms', parameters('serverFarm_name'))]"
+            ],
+            "kind": "app",
+            "properties": {
+                "enabled": true,
+                "hostNameSslStates": [
+                    {
+                        "name": "[concat(parameters('site_name'), '.azurewebsites.net')]",
```
Comment (bmoore-msft) [inline 101-private-endpoint-webapp/azuredeploy.json:114]: remove the hardcoded "azurewebsites.net"

File: 101-private-endpoint-webapp/azuredeploy.json [hunk 4]
```diff
@@ -0,0 +1,224 @@
...
+                        "sslState": "Disabled",
+                        "hostType": "Standard"
+                    },
+                    {
+                        "name": "[concat(parameters('site_name'), '.scm.azurewebsites.net')]",
```
Comment (bmoore-msft) [inline 101-private-endpoint-webapp/azuredeploy.json:119]: remove the hardcoded "azurewebsites.net"

File: 101-private-endpoint-webapp/azuredeploy.json [hunk 5]
```diff
@@ -0,0 +1,224 @@
...
+                "[resourceId('Microsoft.Web/sites', parameters('site_name'))]"
+            ],
+            "properties": {
+                "ftpsState": "AllAllowed"
+            }
+        },
+        {
+            "type": "Microsoft.Web/sites/hostNameBindings",
+            "apiVersion": "2019-08-01",
+            "name": "[concat(parameters('site_name'), '/', parameters('site_name'), '.azurewebsites.net')]",
```
Comment (bmoore-msft) [inline 101-private-endpoint-webapp/azuredeploy.json:142]: remove the hardcoded "azurewebsites.net"

File: 101-private-endpoint-webapp/azuredeploy.json (hunk 1 above)
Comment (Ericgre) [reply 101-private-endpoint-webapp/azuredeploy.json:5]: I added for each parameter a metadata

File: 101-private-endpoint-webapp/azuredeploy.json (hunk 2 above)
Comment (Ericgre) [reply 101-private-endpoint-webapp/azuredeploy.json:27]: already done for non unique value

File: 101-private-endpoint-webapp/azuredeploy.json (hunk 3 above)
Comment (Ericgre) [reply 101-private-endpoint-webapp/azuredeploy.json]: done

File: 101-private-endpoint-webapp/azuredeploy.json (hunk 4 above)
Comment (Ericgre) [reply 101-private-endpoint-webapp/azuredeploy.json]: done

File: 101-private-endpoint-webapp/azuredeploy.json (hunk 5 above)
Comment (Ericgre) [reply 101-private-endpoint-webapp/azuredeploy.json]: done

**Pull request merged**